### ISSUE TITLE
Error when i try collectstatic

### ISSUE THREAD

**Issue by alejandrojnm** [py-pa/django-minio-storage#37]
After install django-minio-storage and configure, i get this error:
```
Django Version: 1.11.4
Python Version: 3.6.2
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'minio_storage',
 'system']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/options.py" in wrapper
  551.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/sites.py" in inner
  224.             return view(request, *args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/options.py" in add_view
  1508.         return self.changeform_view(request, None, form_url, extra_context)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/options.py" in changeform_view
  1408.             return self._changeform_view(request, object_id, form_url, extra_context)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/options.py" in _changeform_view
  1448.                 self.save_model(request, new_object, form, not add)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/contrib/admin/options.py" in save_model
  979.         obj.save()

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/base.py" in save
  807.                        force_update=force_update, update_fields=update_fields)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/base.py" in save_base
  837.             updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/base.py" in _save_table
  923.             result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/base.py" in _do_insert
  962.                                using=using, raw=raw)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/manager.py" in manager_method
  85.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/query.py" in _insert
  1076.         return query.get_compiler(using=using).execute_sql(return_id)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in execute_sql
  1106.             for sql, params in self.as_sql():

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in as_sql
  1059.                 for obj in self.query.objs

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in <listcomp>
  1059.                 for obj in self.query.objs

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in <listcomp>
  1058.                 [self.prepare_value(field, self.pre_save_val(field, obj)) for field in fields]

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/sql/compiler.py" in pre_save_val
  1008.         return field.pre_save(obj, add=True)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/fields/files.py" in pre_save
  296.             file.save(file.name, file.file, save=False)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/db/models/fields/files.py" in save
  94.         self.name = self.storage.save(name, content, max_length=self.field.max_length)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/core/files/storage.py" in save
  53.         name = self.get_available_name(name, max_length=max_length)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/django/core/files/storage.py" in get_available_name
  77.         while self.exists(name) or (max_length and len(name) > max_length):

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/minio_storage/storage.py" in exists
  95.             self.client.stat_object(self.bucket_name, self._sanitize_path(name))

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/minio/api.py" in stat_object
  913.                                   object_name=object_name)

File "/Volumes/HDD2/Proyectos/python/netflix/lib/python3.6/site-packages/minio/api.py" in _url_open
  1774.                                     object_name).get_exception()

Exception Type: NoSuchKey at /admin/system/movie/add/
Exception Value: NoSuchKey: message: The specified key does not exist.
```

i check in minio server via web and the dir static and media is there, any idea.

**Comment by thomasf** [comment]
This is a kown issue,  I will try to resolve these issues it soon so what we can create a new release

related issues:

https://github.com/py-pa/django-minio-storage/issues/35
https://github.com/py-pa/django-minio-storage/pull/25
https://github.com/py-pa/django-minio-storage/pull/36
https://github.com/py-pa/django-minio-storage/issues/29

**Comment by thomasf** [comment]
probably fixed by 67e6be7c6d7031f35fd8a4b65a9d58cc8cc2d105
